testsuite/features/secondary/proxy_as_pod_basic_tests.feature

```diff
@@ -0,0 +1,274 @@
+    And I wait until "uyuni-proxy-tftpd" service is active on "proxy"
+    And I wait until port "8022" is listening on "proxy"
+    And I wait until port "8080" is listening on "proxy"
+    And I wait until port "443" is listening on "proxy"
+    And I visit "Proxy" endpoint of this "proxy"
+
+  Scenario: Containerized Proxy should be registered automatically
+    When I follow the left menu "Systems"
+    And I wait until I see the name of "containerized_proxy", refreshing the page
+
+  Scenario: Remove the offending key in salt known hosts
+    When I remove offending SSH key of "containerized_proxy" at port "8022" for "/var/lib/salt/.ssh/known_hosts" on "server"
+
+  Scenario: Bootstrap a Salt minion in the Containerized Proxy
+    When I follow the left menu "Systems > Bootstrapping"
+    Then I should see a "Bootstrap Minions" text
+    When I enter the hostname of "sle_minion" as "hostname"
+    And I enter "22" as "port"
+    And I enter "root" as "user"
+    And I enter "linux" as "password"
+    And I select the hostname of "containerized_proxy" from "proxies"
+    And I click on "Bootstrap"
+    And I wait until I see "Successfully bootstrapped host!" text
+
+  Scenario: Check the new bootstrapped minion in System Overview page
+    When I follow the left menu "Salt > Keys"
+    And I wait until I do not see "Loading..." text
+    Then I should see a "accepted" text
+    When I follow the left menu "Systems > System List > All"
+    And I wait until I see the name of "sle_minion", refreshing the page
+    And I wait until onboarding is completed for "sle_minion"
+    Then the Salt master can reach "sle_minion"
+
+  Scenario: Check connection from minion to Containerized Proxy
+    Given I am on the Systems overview page of this "sle_minion"
+    When I follow "Details" in the content area
+    And I follow "Connection" in the content area
+    Then I should see "containerized_proxy" short hostname
+
+  Scenario: Check registration on Containerized Proxy of minion
+    When I follow the left menu "Systems > System List > All"
+    And I follow this "containerized_proxy" link
+    And I follow "Details" in the content area
+    And I follow "Proxy" in the content area
+    Then I should see "sle_minion" hostname
+
+  Scenario: Salt minion grains are displayed correctly on the details page
+    Given I am on the Systems overview page of this "sle_minion"
+    Then the hostname for "sle_minion" should be correct
+    And the kernel for "sle_minion" should be correct
+    And the OS version for "sle_minion" should be correct
+    And the IPv4 address for "sle_minion" should be correct
+    And the IPv6 address for "sle_minion" should be correct
+    And the system ID for "sle_minion" should be correct
+    And the system name for "sle_minion" should be correct
+    And the uptime for "sle_minion" should be correct
+    And I should see several text fields for "sle_minion"
+
+  Scenario: Install a patch on the Salt minion
```

Comment:
Some of the Scenario names are to common. You should use something like e.g.
```suggestion
  Scenario: Install a patch on the Salt minion in the Containerized Proxy content
```
The same applies to other scenario names. Remember we want to have unique scenario names at some point.

Reply:
Make sense I will rename then according, thank you.

Reply:
Good idea. Makes totally sense!